spec/system/visiting_map_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "rails_helper"
+
+RSpec.describe "visiting maps", type: :system do
+  it "cannot zoom the map" do
```

Comment:
Maybe a context to specify that this is players rather than DMs?

Reply:
I generally standardize on the word "Visitor" for anyone doing things when they are no logged in. I'll rename this so it's more explicitly "Visitor".

I'd like to hold off on introducing the Player language until we've got actual players, as that might be something specific in the domain. Sound ok?

Reply:
Ah right, so 'visiting' as a verb here refers to an anonymous user?

Happy to avoid referring to players until the concept exists.

Reply:
Yes, but I've changed it to "Visitor views map" to make it more clear.

Reply:
Great, thank you!